src/test-run/index.js

```diff
@@ -433,7 +433,9 @@ export default class TestRun extends AsyncEventEmitter {
     async _enqueueBrowserConsoleMessagesCommand (command, callsite) {
         await this._enqueueCommand(command, callsite);
 
-        return this.consoleMessages.getCopy();
+        const consoleMessageCopy = this.consoleMessages.getCopy();
```

Comment:
`this.consoleMessages.getCopy();` returns dictionary with key `null` in case of non multiple windows mode.
It works since `this.browserConnection.activePageId` also equals null. It is a valid way, but what if we will have `activePageId` even in non multiple windows mode. In this case we can avoid the `null` key.

Reply:
To have a `pageId` we need to change url format, as I understand

Reply:
At present, the `pageId` parameter presents only in multi window mode for local browsers. The paremeter stores in the URL. 
For `headless`, `remote`, `path` and `legacy` browsers we cannot create a proxy url with the `pageId` parameter (it's impossible or need to write an additional code).  For these modes need to have the `null` pageId to store browser console messages.